### PR TITLE
Chore: Kwik "Price Estimate" Update.

### PR DESCRIPTION
**Pull request by Saint-lawal**
### What does this PR do?
- Use "32" (card) for estimate instead of 524288.

### CODE REVIEW

File: shared/helpers/KwikHelpers.js [hunk 1]
```diff
@@ -257,7 +257,7 @@ class KwikHelpers {
     }
   }
 
-  static getBasePricingPayload() {
+  static getBasePricingPayload(isEstimation) {
```
Comment (bolorundurowb) [inline shared/helpers/KwikHelpers.js:260]: You dont seem to be using the value passed here

File: shared/helpers/KwikHelpers.js [hunk 2]
```diff
@@ -271,7 +271,7 @@ class KwikHelpers {
       is_schedule_task: 0,
       has_pickup: 1,
       has_delivery: 1,
-      payment_method: 524288, // 524288  is for end  of the month payments
+      payment_method: 32, // 524288  is for end  of the month payments
```
Comment (bolorundurowb) [inline shared/helpers/KwikHelpers.js:274]: Can we update the comment to explain the value

File: shared/helpers/KwikHelpers.js (hunk 1 above)
Comment (Saint-lawal) [reply shared/helpers/KwikHelpers.js]: Nice, forgot to remove this. Done 👍

File: shared/helpers/KwikHelpers.js (hunk 2 above)
Comment (Saint-lawal) [reply shared/helpers/KwikHelpers.js]: Done

**Review by bolorundurowb — APPROVED** [review]

**Pull request merged**